willibald_dwh/macros/psa_view.sql

```diff
@@ -0,0 +1,22 @@
+
+{% macro psa_view(p_psa_table) %}
+
+{% if execute %}
+
+  {% set v_psa_table_node = graph.nodes.values()
+     | selectattr("resource_type", "equalto", "seed")
+     | selectattr("name", "equalto", p_psa_table)
+     | first %}
+
+{% endif %}
+  
+
+SELECT
+    sys_loadingid,
+    {% if execute %}LEAD(sys_loadingid) OVER (PARTITION BY {{ v_psa_table_node.meta.unique_key }} ORDER BY sys_loadingid ASC) IS NULL AS sys_islatest,{% endif %}
```

Comment:
Refactor this so that a variable is defined with a default Value of 1, which will be used if no Metadata can be retrieved.